[OSDOCS#12997]: Update agent-based installer documentation to include 2NO (#110741)
openshift/openshift-docs: modules/understanding-agent-install.adoc

Reviewer: Please revert every change in this file except for the change necessary to document your feature

|====
|Topology|Number of control plane nodes|Number of compute nodes|vCPU|Memory|Storage
|Single-node cluster|1|0|8 vCPUs|16 GB of RAM| 120 GB

> |Two-Node with Fencing (TNF)|2|0|4 vCPUs|16 GB of RAM|120 GB

Reviewer 2: Two-Node with Arbiter is going to need two lines here:

|Two-Node with Arbiter (standard Control Plane nodes)|2|0|4 vCPUs|16 GB of RAM|120 GB
|Two-Node with Arbiter (Arbiter node|1|0|2 vCPUs|8 GB of RAM|50 GB

|Single-node cluster|1|0|8 vCPUs|16 GB of RAM| 120 GB
|Compact cluster|3|0 or 1|8 vCPUs|16 GB of RAM|120 GB
|HA cluster|3 to 5|2 and above |8 vCPUs|16 GB of RAM|120 GB

> |Two-Node with Fencing (TNF)|2|0|4 vCPUs|16 GB of RAM|120 GB

Reviewer 3: Show we care about ordering here? I would go:
Single-Node
Two-Node with Fencing
Two-Node with Arbiter (if we need that)
Compact cluster (i.e. 3 CP nodes, no workers)
HA Cluster (3 CP nodes with workers or 5 CP nodes usually also with workers)

Author: Just need to add TNA

* `external`
* `none`

> +

Reviewer: This plus sign should be replaced with a blank line, since it is causing the line below to be incorrectly indented:

```suggestion

```

> For two-node OpenShift cluster with fencing (TNF), only the following platforms are supported:

Reviewer: ```suggestion
For a two-node {product-title} cluster with fencing (TNF), only the following platforms are supported:
```

Or you can keep out {product-title} and just say "two-node cluster"

* baremetal
* external

> * none

Reviewer: These need to be in backticks like the list above:

```suggestion
* `baremetal`
* `external`
* `none`
```
